modules/infra/control/ctlplane.c

```
@@ -385,33 +383,23 @@ static void cp_update(struct iface *iface) {
+			    strerror(errno));
 	}
 
-	memccpy(ifr.ifr_name, iface->name, 0, IFNAMSIZ);
-	ifr.ifr_mtu = iface->mtu;
-
-	if (ioctl(ioctl_sock, SIOCSIFMTU, &ifr) < 0) {
-		LOG(ERR, "ioctl(SIOCSIFMTU) %s", strerror(errno));
-		goto err;
+	if (iface_get_eth_addr(iface, &mac) == 0) {
+		if (netlink_link_set_mac(iface->cp_id, &mac) < 0)
+			LOG(ERR,
```

Cppcheck (2.18.0)

[error] There is an unknown macro here somewhere. Configuration is required. If ETH_F is a macro then please configure it.

(unknownMacro)